web/api/serializers.py

```
@@ -85,37 +83,30 @@ class Meta:
 		fields = '__all__'
 		depth = 2
 
-	@staticmethod
-	def get_vuln_count(obj):
+	def get_vuln_count(self, obj):
 		try:
 			return obj.vuln_count
 		except:
```

Ruff

Do not use bare `except`

(E722)